packages/safe-tools-client/app/components/future-payments-list/scheduled-payment-card/index.css

```diff
@@ -0,0 +1,55 @@
+.scheduled-payment-card {
+  display: block;
+
+  padding: var(--boxel-sp-sm) var(--boxel-sp);
+  margin-bottom: var(--boxel-sp-sm);
+}
+
+.scheduled-payment-card .boxel-icon-button {
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+.scheduled-payment-card__content {
+  display: flex;
+  flex-direction: column;
+}
+
+.scheduled-payment-card_content-payment-detail {
```

Comment:
In these class names in this file you are using both `__` and `_`. Did you mean to use `__`?